src/methods/combinations-with-replacement/combinations-with-replacement.mjs

```diff
@@ -16,20 +16,19 @@ function isSorted(arr) {
 }
 
 export function combinationsWithReplacement(iterable, r) {
-  const arr = Array.from(iterable);
+  const arr = [...iterable];
```

Comment:
Why do you use spread operator? `Array.from` was chosen because babel does not fill resulting code with helper functions.

Reply:
Ah, I hadn't been thinking of that. I would have hoped that babel would just transpile it to `Array.from`. I just put it that way because in the long run I think language features are preferable to builtins.

Reply:
Actually to be honest I'm almost certain I prefer the babel helper here. `Array.from` is tricky. Some platforms/browsers implemented `Symbol.iterator` before they implemented `Array.from`. Others implemented them in the reverse order, meaning some versions of `Array.from` in the wild *may not actually support iterables*. Contrast that with the babel helper, which I am certain always does the same thing. And the helper is a module, so there should only be one copy of it across the entire project. Given that we use array spread in other code, it's already going to be part of the bundle anyway.

Reply:
And I believe that was the last `Array.from` outside test code, so now the library no longer needs it to exist/be implemented correctly at all to work.